specification/storage/resource-manager/Microsoft.Storage/storageDataRP/preview/2024-11-01-preview/connectors.json

```diff
@@ -0,0 +1,1065 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Storage Data Resource Provider",
+    "version": "2024-11-01-preview",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
```

Comment:
Do u have an exception for not being on RPaaS? If so, can u provide some evidence of the same? Please send the email to rmanda and the armapireview alias.